README.md

```diff
@@ -284,8 +286,16 @@ Example usage:
     '{Recently Delivered} = TRUE()',
     [{ field: 'Full Name' }]
   );
-
+```
 
 ## Notes
 
-This generator was made originally to support Blueprint projects. Learn more about what we do here: https://calblueprint.org/
+This generator was made originally to support Blueprint projects. Learn more about what we do here: <https://calblueprint.org/>
+
+## Known issues
+
+### Schema change errors
+
+In version 1.5.1 and earlier, the generated `airtable.js` would throw errors upon being unable to properly transform the Airtable call internally. It uses the local copy of `schema.js` to perform the transformations, which unfortunately caused client applications to break upon schema changes until the local copy of `schema.js` was updated - i.e the generator was re-run.
```

Comment:
Wow this is so professionally written 👏

Reply:
Also such a high-leverage change LOL

Reply:
in this ~~house~~ repo we stan documentation 😂 

I updated it to include the cases I can think of, I'll try to fix the rest later 😔